test/db/asm/mips32

```diff
@@ -0,0 +1,59 @@
+d "nop" 00000000 0x4
```

Comment:
please rename this file as `mips_32` because this `_32` will be used to set the bits to `32`